specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/firewallPolicy.json

```diff
@@ -622,10 +626,84 @@
         "intrusionSystemMode": {
           "description": "The operation mode for Intrusion system.",
           "$ref": "#/definitions/FirewallPolicyIntrusionSystemMode"
+        },
+        "transportSecurity": {
+          "description": "TLS Configuration definition.",
+          "$ref": "#/definitions/FirewallPolicyTransportSecurity"
         }
       },
       "description": "Firewall Policy definition."
     },
+    "FirewallPolicyTransportSecurity": {
+      "properties": {
+        "certificatesAuthority": {
```

Comment:
>s [](start = 20, length = 1)

This looks like a typo and should be `certificateAuthority`

Reply:
@majastrz, @number213 , done.